base64.h warnings reported with use_lto=1

I see following warning:
```
./thirdparty/misc/base64.h:14: warning: type of 'base64_encode' does not match original declaration [-Wlto-type-mismatch]
   14 | uint32_t base64_encode(char *to, char *from, uint32_t len);
      | 
thirdparty/misc/base64.c:14: note: return value type mismatch
   14 | long base64_encode (to, from, len)
      | 
thirdparty/misc/base64.c:14: note: type 'long int' should match type 'uint32_t'
thirdparty/misc/base64.c:14: note: 'base64_encode' was previously declared here
thirdparty/misc/base64.c:14: note: code may be misoptimized unless -fno-strict-aliasing is used
./thirdparty/misc/base64.h:15: warning: type of 'base64_decode' does not match original declaration [-Wlto-type-mismatch]
   15 | uint32_t base64_decode(char *to, char *from, uint32_t len);
      | 
thirdparty/misc/base64.c:66: note: return value type mismatch
   66 | long base64_decode (to, from, len)
      | 
thirdparty/misc/base64.c:66: note: type 'long int' should match type 'uint32_t'
thirdparty/misc/base64.c:66: note: 'base64_decode' was previously declared here
thirdparty/misc/base64.c:66: note: code may be misoptimized unless -fno-strict-aliasing is used
```

The error looks correct, it's really a type mismatch.

--- a/thirdparty/misc/base64.h
+++ b/thirdparty/misc/base64.h
@@ -11,8 +11,8 @@
 
 extern "C" {
 
-uint32_t base64_encode(char *to, char *from, uint32_t len);
-uint32_t base64_decode(char *to, char *from, uint32_t len);
+long base64_encode(char *to, char *from, unsigned int len);
+long base64_decode(char *to, char *from, unsigned int len);
 };
 
 #endif /* BASE64_H */

